cel/cel_custom.c

```diff
@@ -103,7 +103,12 @@ static int load_config(void)
 			}
 
 			ast_string_field_build(sink, format, "%s\n", var->value);
-			ast_string_field_build(sink, filename, "%s/%s/%s", ast_config_AST_LOG_DIR, name, var->name);
+			if (var->name[0] == '/') {
+				ast_string_field_build(sink, filename, "%s", var->name);
+			}
+			else {
```

Comment:
This should all be on one line according to [the Coding Guidelines](https://docs.asterisk.org/Development/Policies-and-Procedures/Coding-Guidelines/#if-statements) (which is coincidentally not rendering properly on the docs site):

```
} else {
```